es/CHANGELOG.md

```diff
@@ -21,8 +21,8 @@
 
 26 de octubre de 2020: 🎊 el sexto borrador parte un requisito y añade claridad.
 
-* Divide el criterio «Las colaboradoraciones son bienvenidas» ("Welcome contributors") en «Hacer que las contribuciones sean fáciles» ("Make contributing easy") y «Las colaboradoraciones son bienvenidas» ("Welcome contributors").
-* Cambia el nombre del criterio «Prestar atención a la madurez de la codebase» ("Pay attention to codebase maturity") por «Documentar la madurez de la codebase» ("Document codebase maturity").
+* Divide el criterio «Las colaboraciones son bienvenidas» ("Welcome contributors") en «Hacer que las contribuciones sean fáciles» ("Make contributing easy") y «Las colaboraciones son bienvenidas» ("Welcome contributors").
+* Cambia el nombre del criterio «Prestar atención a la madurez del codebase» ("Pay attention to codebase maturity") por «Documentar la madurez del codebase» ("Document codebase maturity").
```

Comment:
This is a tough one but I believe I'm going to stick to my version.

The feminine article makes more sense to me because I translate "codebase" as "base de código" and "base" is feminine. I've found some documentation from José Ramón Sampedro, a former academic from the Royal Spanish Academy (of the Spanish language) in which he also supports that way of reasoning. You can find it (here)[http://www2.ati.es/gt//lengua-informatica/externos/sampedr1.html#angli1]